### PR TITLE
Add support for Edge load_statistics.db

### PR DESCRIPTION
**Pull request by trashg0blin**
## One line description of pull request
Adding support for Edge load_statistics.db

## Description:
Edge load_statistics is an anti-tracking mechanism that can be used to identify embedded resource load events within a page that can be combined with browser history for an accurate picture of navigation activities. Useful for identifying phishkit related activity.

**Related issue (if applicable):**
Enhancement for #4732 

## Notes:
All contributions to Plaso undergo [code review](https://github.com/log2timeline/l2tdocs/blob/main/process/Code%20review%20process.asciidoc).
This makes sure that the code has appropriate test coverage and conforms to the
[Plaso style guide](https://plaso.readthedocs.io/en/latest/sources/developer/Style-guide.html).

One of the maintainers will examine your code, and may request changes. Check off the items below in
order, and then a maintainer will review your code.

## Checklist:
* [x] Automated checks (GitHub Actions, AppVeyor) pass
* [x] No new [new dependencies](https://plaso.readthedocs.io/en/latest/sources/developer/Adding-a-new-dependency.html) are required or l2tdevtools has been updated
* [x] Reviewer assigned

### CODE REVIEW

**Comment by trashg0blin** [comment]
Fixed linting issues with /data/formatters/browser.yaml

**Comment by joachimmetz** [comment]
Note that I was not able to get around to this yet, but I've not forgotten about it

File: plaso/lib/definitions.py [hunk 1]
```diff
@@ -143,6 +143,7 @@
 TIME_DESCRIPTION_LAST_RUN = 'Last Time Executed'
 TIME_DESCRIPTION_LAST_SEEN = 'Last Seen Time'
 TIME_DESCRIPTION_LAST_SHUTDOWN = 'Last Shutdown Time'
+TIME_DESCRIPTION_LAST_UPDATED= 'Last Update Time'
```
Comment (joachimmetz) [inline plaso/lib/definitions.py]: nit: missing space before `=`

**Review by joachimmetz — APPROVED** [review]
LGTM

**Pull request merged**